bugzilla/base.py

```diff
@@ -266,6 +266,7 @@ def __init__(self, url=-1, user=None, password=None, cookiefile=-1,
             to file or directory for custom certs.
         :param api_key: A bugzilla5+ API key
         :param configpaths: A list of possible bugzillarc locations.
+        :param basic_auth: Use headers with basic authentication
```

Comment:
Please replace references of 'basic authentication' with 'HTTP Basic authentication' which is more googleable, personally I didn't know what this all meant at first